grpc_server/README.md

```diff
@@ -321,7 +335,7 @@ iex> {:ok, reply} = channel |> Helloworld.GreetingServer.Stub.say_unary_hello(re
 
 ## Target Schemes and Resolvers
 
-The `connect/2` function supports URI-like targets that are resolved via the internal **gRPC** [Resolver](lib/grpc/client/resolver.ex).  
+The `connect/2` function supports URI-like targets that are resolved via the internal **gRPC** [Resolver](lib/grpc/client/resolver.ex).
```

Comment:
This link still points to the old location; it needs to be moved to ../grpc_client/lib/grpc/client/resolver.ex